### PR TITLE
Feature: Volkswagen MEB platform support

### PR DESCRIPTION
**Pull request by dalathegreat**
### What
This PR adds basic support for [MEB platform](https://github.com/dalathegreat/Battery-Emulator/wiki/MEB-battery)

### Why
To increase the amount of batteries supported by the emulator, making the software more useful

### How
Check the [MEB platform](https://github.com/dalathegreat/Battery-Emulator/wiki/MEB-battery) wiki page for more info on how to set up a MEB battery

### CODE REVIEW

**Comment by mvgalen** [comment]
> ### What
> This PR adds basic support for MEB platform

Hi Dala,

What is the prospect on this feature branch. Does it seem promising? I am able to buy a id3 battery for a decent price, but would want to use it with your emulator. I am able to help in debugging.

Br,
Marijn

**Comment by TheGreenJosip** [comment]
> > ### What
> > This PR adds basic support for MEB platform
> 
> Hi Dala,
> 
> What is the prospect on this feature branch. Does it seem promising? I am able to buy a id3 battery for a decent price, but would want to use it with your emulator. I am able to help in debugging.
> 
> Br, Marijn

Yes I would also be really interested and am able to provide any help as I have the lilygo and battery pack (12S one) already at home. 🙏🏽

**Comment by mvgalen** [comment]
> > > ### What
> > > This PR adds basic support for MEB platform
> > 
> > 
> > Hi Dala,
> > What is the prospect on this feature branch. Does it seem promising? I am able to buy a id3 battery for a decent price, but would want to use it with your emulator. I am able to help in debugging.
> > Br, Marijn
> 
> Yes I would also be really interested and am able to provide any help as I have the lilygo and battery pack (12S one) already at home. 🙏🏽

Best way forward I guess if you want to help is become Patreon supporter and join the discord channel. The cangesof the commits of today originated there.

**Comment by TheGreenJosip** [comment]
Is any more help for supporting the MEB-Modules needed in any way? How is the implementation going so far?

**Comment by dalathegreat** [comment]
@TheGreenJosip This is very offtopic, please create a new issue for your question and we will answer from there

File: Software/Software.ino [hunk 1]
```diff
@@ -257,6 +258,20 @@ void core_loop(void* task_time_us) {
     receive_can_native();  // Receive CAN messages from native CAN port
 #ifdef CAN_FD
     receive_canfd();  // Receive CAN-FD messages.
+    if (millis() - previousMillis1s >= INTERVAL_1_S) {
+      previousMillis10ms = millis();
```
Comment (dalathegreat) [inline Software/Software.ino:262]: Typo, you are resetting 10ms variable here, think you intended to have the 1s here?

**Pull request merged**